docs/about.md

```diff
@@ -71,6 +73,8 @@ The result is a market that can function well when trading volume is low without
 
 Read **[our technical guide](https://www.notion.so/Technical-Guide-to-Manifold-Markets-b9b48a09ea1f45b88d991231171730c5)** to find out more!
 
+Binary markets created after March 15 use a variation on constant-product market makers due to Manifold user Pepe: Instead of the product (or equivalently, the geometric mean) of YES shares and NO shares on the table, they hold constant the weighted geometric mean, with weights equal to the probabilities chosen by the market creator at creation.
```

Comment:
Would reword as:

"Newer binary markets use a constant function market maker, by the formula .... This hold constant the weighted geometric mean, with weights equal to the probabilities chosen by the market creator at creation.

This design was inspired by Uniswap's CPMM and a suggestion from Manifold user Pepe" (and then link to the appropriate places).

Reply:
Reasons:
- Focus first on what these are, because that's what the user cares about
- Assign credit at the end, not as an aide